### PR TITLE
Clarify consistency in table and blockchain docs

### PR DESCRIPTION
**Pull request by vegarsti**
Closes PRO-1332.

### CODE REVIEW

File: api-reference/tables/blockchain/insert.mdx [hunk 1]
```diff
@@ -123,6 +123,16 @@ On the initial insert, the endpoint will automatically create a table for you un
 - The maximum allowable request size is 1.2GB.
 </Note>
 
+## Consistency
+A request to this endpoint has two possible outcomes: Either all of the data in the request was inserted, or none of it was.
+It's not possible for parts of the request data to be inserted, while the rest is not inserted.
+In other words, plase use and trust the status codes that the endpoint returns.
```
Comment (helanto) [inline api-reference/tables/blockchain/insert.mdx:129]: plase 👉 please

Comment (vegarsti) [reply api-reference/tables/blockchain/insert.mdx:129]: Thanks!

**Comment by vegarsti** [comment]
Good catch! Does it look good otherwise? @helanto

**Review by helanto — APPROVED** [review]

File: api-reference/tables/blockchain/insert.mdx [hunk 2]
```diff
@@ -123,6 +123,16 @@ On the initial insert, the endpoint will automatically create a table for you un
...
 </Note>
 
+## Consistency
+A request to this endpoint has two possible outcomes: Either all of the data in the request was inserted, or none of it was.
+It's not possible for parts of the request data to be inserted, while the rest is not inserted.
+In other words, please use and trust the status codes that the endpoint returns.
+A status code of 200 means that the data in the request was successfully inserted.
+If you get any other status code, you can safely retry your request after addressing the issue that the error message indicated.
+
+## Concurrent requests
```
Comment (hawkaa) [inline api-reference/tables/blockchain/insert.mdx:133]: This should go into the `table` part too.

Can you see if it is possible to reuse the same text by splitting it out to a third file and including it as a kind of template?

Comment (vegarsti) [reply api-reference/tables/blockchain/insert.mdx:133]: It is in the table part!

But good idea!

Comment (hawkaa) [reply api-reference/tables/blockchain/insert.mdx:133]: Ah, I didn't see it was there from before. Sorry!

**Pull request merged**